ecs-agent/utils/netlinkwrapper/netlink_linux.go

```diff
@@ -38,3 +40,11 @@ func (nl *netLink) LinkByName(name string) (netlink.Link, error) {
 func (nl *netLink) LinkSetUp(link netlink.Link) error {
 	return netlink.LinkSetUp(link)
 }
+
```

Comment:
nit (non-blocking): maybe we can add a comment here saying this is equivalent to running `ip route show`

Reply:
Yes added.